recipes/qt/5.x.x/conanfile.py

```diff
@@ -662,6 +666,46 @@ def package(self):
         for fl in glob.glob(os.path.join(self.package_folder, "lib", "*Qt5Bootstrap*")):
             os.remove(fl)
 
+        for m in os.listdir(os.path.join(self.package_folder, "lib", "cmake")):
+            module = os.path.join(self.package_folder, "lib", "cmake", m, "%sMacros.cmake" % m)
+            if not os.path.isfile(module):
+                tools.rmdir(os.path.join(self.package_folder, "lib", "cmake", m))
+
+
+        extension = ""
+        if self.settings.os == "Windows":
+            extension = ".exe"
+        v = tools.Version(self.version)
+        filecontents = textwrap.dedent("""\
+            set(QT_CMAKE_EXPORT_NAMESPACE Qt5)
+            set(QT_VERSION_MAJOR {major})
+            set(QT_VERSION_MINOR {minor})
+            set(QT_VERSION_PATCH {patch})
+        """.format(major=v.major, minor=v.minor, patch=v.patch))
+        targets = {}
+        targets["Core"] = ["moc", "rcc", "qmake"]
+        targets["DBus"] = ["qdbuscpp2xml", "qdbusxml2cpp"]
+        if self.options.widgets:
+            targets["Widgets"] = ["uic"]
+        if self.options.qttools:
+            targets["Tools"] = ["qhelpgenerator", "qcollectiongenerator", "qdoc", "qtattributionsscanner"]
+            targets[""] = ["lconvert", "lrelease", "lupdate"]
```

Comment:
What happen with this empty key?

Reply:
it's not an empty key, it's just a string with zero characters. it's used to create these https://github.com/qt/qttools/blob/v5.15.2/src/linguist/Qt5LinguistToolsConfig.cmake.in#L89-L90 (I have no idea why qt is inconsistent here)